rd_ui/app/scripts/controllers/dashboard.js

```diff
@@ -32,22 +32,23 @@
                 // TODO: first object should be a copy, otherwise one of the chart filters behaves different than the others.
                 filters[filter.name] = filter;
                 filters[filter.name].originFilters = [];
+                if (_.has($location.search(), filter.friendlyName)) {
```

Comment:
How about we use `filter.name`? It has more chances to be more URL friendly (in most cases it's the table column name -> no capital letters and no spaces).